lldp_8021qaz.c

```diff
@@ -210,6 +210,7 @@ static int read_cfg_file(char *ifname, struct lldp_agent *agent,
 	char arg_path[256];
 	int res = 0, i;
 	int willing, pfc_mask, delay;
+	char *parse;
```

Comment:
It might be good to initialize this to `NULL` in the unlikely case that the loop never reaches `strdup()` during any iterations.